syft/frameworks/torch/tensors/pointer.py

```diff
@@ -2,19 +2,21 @@
 
 
 class PointerTensor(AbstractTensor):
-    """A pointer to another tensor. A PointerTensor forwards all API calls to the remote.
-
-    PointerTensor objects point to tensors (as their name implies). They exist to mimic
-    the entire API of a normal tensor, but instead of computing a tensor function locally
-    (such as addition, subtraction, etc.) they forward the computation to a remote machine
-    as specified by self.location. Specifically, every PointerTensor has a tensor
-    located somewhere that it points to (they should never exist by themselves). Note that
-    PointerTensor objects can point to both torch.Tensor objects AND to other PointerTensor
-    objects. Furthermore, the objects being pointed to can be on the same machine or
-    (more commonly) on a different one. Note further that a PointerTensor does not know
-    the nature how it sends messages to the tensor it points to (whether over socket, http,
-    or some other protocol) as that functionality is abstracted in the BaseWorker object in
-    self.location.
+    """A pointer to another tensor.
+
+    A PointerTensor forwards all API calls to the remote.PointerTensor objects
+    point to tensors (as their name implies). They exist to mimic the entire
+    API of a normal tensor, but instead of computing a tensor function locally
+    (such as addition, subtraction, etc.) they forward the computation to a
+    remote machine as specified by self.location. Specifically, every
+    PointerTensor has a tensor located somewhere that it points to (they should
+    never exist by themselves). Note that PointerTensor objects can point to
+    both torch.Tensor objects AND to other PointerTensor objects. Furthermore,
+    the objects being pointed to can be on the same machine or (more commonly)
+    on a different one. Note further that a PointerTensor does not know the
+    nature how it sends messages to the tensor it points to (whether over
+    socket, http, or some other protocol) as that functionality is abstracted
+    in the BaseWorker object in self.location.
 
     :Example:
```

Comment:
Following other PRs on this issue should these `:Example:` sections be turned into something like `Example:` (indented 4 spaces). I'm looking at PR #1798 and was wondering what's a good way to add these examples in the comment. Not sure the google style guide talks about this in particular.

Reply:
Agree, its changed to Example: (indented 4 spaces). I am too not sure what is the best way to add examples in the docstring. Although, Example: (indented 4 spaces) looks readable and more in line with others such as Args, Returns etc.